trulens_eval/trulens_eval/tru_llama.py

```diff
@@ -251,6 +251,13 @@ def select_source_nodes(cls) -> Lens:
         """
         return cls.select_outputs().source_nodes[:]
 
+    @classmethod
+    def select_context(cls) -> Lens:
+        """
+        Get the path to the context in the query output.
+        """
+        return cls.select_outputs().source_nodes[:].node.text
```

Comment:
reuse `select_source_nodes` here?

Reply:
Is there any benefit of keeping this? Or just remove it in place of app.select_context()